api/local/getProgress.ts

```diff
@@ -2,12 +2,14 @@ import { defaultProgressContext } from 'providers/ProgressProvider'
 
 export default async function getProgressLocal(): Promise<string> {
   try {
-    const res = localStorage.getItem('SavingSatoshiProgress')
+    const progress = localStorage.getItem('SavingSatoshiProgress')
 
-    if (!res) {
+    if (!progress) {
       throw new Error('Could not read progress from LocalStorage')
     }
 
+    const res = progress.replace(/['"]/g, '')
```

Comment:
Wasn't this removed in a previous PR?

Reply:
Yes, this is fixed in #452.

This commit was a temporary fix to avoid crashing PR due to an unrelated bug.
I will remove this with the next update of PR.

Reply:
Addressed in 1c7cd6cb913ce1305fa02df9af0a4e5b9c955001

Thanks!